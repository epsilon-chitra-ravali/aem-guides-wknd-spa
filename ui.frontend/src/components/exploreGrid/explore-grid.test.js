import React from 'react';
import ExploreGrid from './index';
import { render, cleanup } from "@testing-library/react";
import '@testing-library/jest-dom'

describe('DSTGridviewMicroFE test', () => {
  let renderedElem

  beforeEach(() => {
    renderedElem = render(<ExploreGrid />);
  });

  afterEach(() => {
    cleanup();
  })

  it('should match Snapshot', async () => {
    expect(renderedElem).toMatchSnapshot('s1');
  });
});