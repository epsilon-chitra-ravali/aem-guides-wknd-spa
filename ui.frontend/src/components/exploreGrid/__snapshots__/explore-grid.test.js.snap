// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DSTGridviewMicroFE test should match Snapshot: s1 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bg-dark bg-opacity-25 container-fluid"
        style="padding-bottom: 5rem;"
      >
        <div
          class="gridCarouselHeader"
        >
          <h2>
            Explore Before You Visit
          </h2>
          <h5>
            Dive deeper into India's rich culture and heritage
          </h5>
        </div>
        <div
          class="h-100 justify-content-center align-items-center hstack gap-3 minBreakpoint-xs"
        >
          <div
            class="card"
            style="width: 18rem; height: 17rem;"
          >
            <div
              class="card-body"
            >
              <img
                alt="image"
                class="d-block w-100"
                loading="lazy"
                src="khajuraho.jpg"
                style="height: 200px;"
              />
              <div
                class="card-title h5"
              >
                khajuraho
              </div>
              <img
                alt="image"
                loading="lazy"
                src="play.png"
                style="height: 40px; width: 40px;"
              />
            </div>
          </div>
          <div
            class="card"
            style="width: 18rem; height: 17rem;"
          >
            <div
              class="card-body"
            >
              <img
                alt="image"
                class="d-block w-100"
                loading="lazy"
                src="taj.jpg"
                style="height: 200px;"
              />
              <div
                class="card-title h5"
              >
                tajmahal
              </div>
              <img
                alt="image"
                loading="lazy"
                src="play.png"
                style="height: 40px; width: 40px;"
              />
            </div>
          </div>
          <div
            class="card"
            style="width: 18rem; height: 17rem;"
          >
            <div
              class="card-body"
            >
              <img
                alt="image"
                class="d-block w-100"
                loading="lazy"
                src="ajanta.jpg"
                style="height: 200px;"
              />
              <div
                class="card-title h5"
              >
                ajanta
              </div>
              <img
                alt="image"
                loading="lazy"
                src="play.png"
                style="height: 40px; width: 40px;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bg-dark bg-opacity-25 container-fluid"
      style="padding-bottom: 5rem;"
    >
      <div
        class="gridCarouselHeader"
      >
        <h2>
          Explore Before You Visit
        </h2>
        <h5>
          Dive deeper into India's rich culture and heritage
        </h5>
      </div>
      <div
        class="h-100 justify-content-center align-items-center hstack gap-3 minBreakpoint-xs"
      >
        <div
          class="card"
          style="width: 18rem; height: 17rem;"
        >
          <div
            class="card-body"
          >
            <img
              alt="image"
              class="d-block w-100"
              loading="lazy"
              src="khajuraho.jpg"
              style="height: 200px;"
            />
            <div
              class="card-title h5"
            >
              khajuraho
            </div>
            <img
              alt="image"
              loading="lazy"
              src="play.png"
              style="height: 40px; width: 40px;"
            />
          </div>
        </div>
        <div
          class="card"
          style="width: 18rem; height: 17rem;"
        >
          <div
            class="card-body"
          >
            <img
              alt="image"
              class="d-block w-100"
              loading="lazy"
              src="taj.jpg"
              style="height: 200px;"
            />
            <div
              class="card-title h5"
            >
              tajmahal
            </div>
            <img
              alt="image"
              loading="lazy"
              src="play.png"
              style="height: 40px; width: 40px;"
            />
          </div>
        </div>
        <div
          class="card"
          style="width: 18rem; height: 17rem;"
        >
          <div
            class="card-body"
          >
            <img
              alt="image"
              class="d-block w-100"
              loading="lazy"
              src="ajanta.jpg"
              style="height: 200px;"
            />
            <div
              class="card-title h5"
            >
              ajanta
            </div>
            <img
              alt="image"
              loading="lazy"
              src="play.png"
              style="height: 40px; width: 40px;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
